Framework/Algorithms/src/CorrectTOFAxis.cpp

```diff
@@ -338,13 +339,18 @@ void CorrectTOFAxis::useReferenceWorkspace(const API::MatrixWorkspace_sptr &outp
 void CorrectTOFAxis::correctManually(const API::MatrixWorkspace_sptr &outputWs) {
   const auto &spectrumInfo = m_inputWs->spectrumInfo();
   const double l1 = spectrumInfo.l1();
-  double l2 = 0;
-  double epp = 0;
+  auto l2 = 0.0;
+  auto epp = 0.0;
+  auto const elasticBinIndexOffset =
+      static_cast<double>(getProperty(PropertyNames::ELASTIC_BIN_INDEX)) - static_cast<int>(m_elasticBinIndex);
```

Comment:
Here, `m_elasticBinIndex` is a conversion in workspace index of the integer part of the property `PropertyNames::ELASTIC_BIN_INDEX`. Can you still do this operation?